app/bundles/CoreBundle/Command/PurgeStaleNotificationsCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Command;
+
+use DateTime;
+use Doctrine\ORM\EntityManager;
+use Mautic\CoreBundle\Entity\Notification;
+use Mautic\CoreBundle\Entity\NotificationRepository;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class PurgeStaleNotificationsCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('mautic:notifications:purge')
+            ->setDescription("Purge stale users' notfications.")
+            ->setDefinition([
+                new InputOption(
+                    'stale-days',
+                    'd',
```

Comment:
D and R are too close together on the keyboard. Screw shortcuts in this case.

Spent the extra second typing out the full param, but remove the risk of screwing up real bad.